src/test/java/com/google/api/codegen/testdata/nodejs/nodejs_multiple_services.baseline

```diff
@@ -216,177 +123,171 @@ module.exports.v1.types = v1Protos;
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
- * EDITING INSTRUCTIONS
- * This file was generated from the file
- * https://github.com/googleapis/googleapis/blob/master/multiple_services.proto,
- * and updates to that file get reflected here through a refresh process.
- * For the short term, the refresh process will only be runnable by Google
- * engineers.
- *
- * The only allowed edits are to method and file documentation. A 3-way
- * merge preserves those additions if the generated source changes.
  */
-'use strict';
 
-var configData = require('./decrementer_service_client_config');
-var extend = require('extend');
-var gax = require('google-gax');
-var path = require('path');
-
-var SERVICE_ADDRESS = 'no-path-templates.googleapis.com';
-
-var DEFAULT_SERVICE_PORT = 443;
-
-var CODE_GEN_NAME_VERSION = 'gapic/0.0.5';
+'use strict';
 
-/*!
- * The scopes needed to make gRPC calls to all of the methods defined in
- * this service.
- */
-var ALL_SCOPES = [
-];
+const gapicConfig = require('./decrementer_service_client_config');
+const gax = require('google-gax');
+const merge = require('lodash.merge');
+const path = require('path');
 
 /**
  *
  * @class
  */
-function DecrementerServiceClient(gaxGrpc, loadedProtos, opts) {
-  opts = extend({
-    servicePath: SERVICE_ADDRESS,
-    port: DEFAULT_SERVICE_PORT,
-    clientConfig: {}
-  }, opts);
-
-  var googleApiClient = [
-    'gl-node/' + process.versions.node
-  ];
-  if (opts.libName && opts.libVersion) {
-    googleApiClient.push(opts.libName + '/' + opts.libVersion);
-  }
-  googleApiClient.push(
-    CODE_GEN_NAME_VERSION,
-    'gax/' + gax.version,
-    'grpc/' + gaxGrpc.grpcVersion
-  );
+class DecrementerServiceClient {
+  constructor(opts) {
+    // Ensure that options include the service address and port.
+    opts = Object.assign({
+      clientConfig: {},
+      port: this.constructor.port,
+      servicePath: this.constructor.servicePath,
+    }, opts);
+
+    // Create a `gaxGrpc` object, with any grpc-specific options
+    // sent to the client.
+    Object.assign(opts, {scopes: this.constructor.scopes});
```

Comment:
I think just assigning opts.scopes directly would be easier.

Reply:
Agreed; done.